S/soapyrtlsdr/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "soapyrtlsdr"
+version = v"0.3.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/pothosware/SoapyRTLSDR.git", "e4b963926012399904aceb57690df3a4f293ce67")
+]
+
+dependencies = [
+    Dependency("librtlsdr_jll"; compat="0.6.0"),
+    Dependency("soapysdr_jll"; compat="0.8.0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd SoapyRTLSDR
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      ..
+make -j${nproc}
+make install
+if [[ "${target}" == *-apple-* ]]; then
+    mv ${libdir}/libyosys.so  ${libdir}/libyosys.dylib
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> arch(p) != "armv6l", supported_platforms(;experimental=true))
+
```

Comment:
```suggestion
platforms = expand_cxxstring_abis(platforms)

```

Reply:
Not sure if you are testing me or not :P, but I don't think this is needed. I just renamed to ".dylib". In the squashed commit I forgot to commit the path change from my copied reference, so I think we are good now.

Reply:
It is needed, see the logs: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=13271&view=results 🙂